scripts/image-builder/build.sh

```diff
@@ -105,30 +107,46 @@ build_image() {
     local image_type=$4
     local parent_blueprint=$5
     local parent_version=$6
+    local buildid
 
     title "Loading ${blueprint} blueprint v${version}"
-    sudo composer-cli blueprints delete ${blueprint} 2>/dev/null || true
+    sudo composer-cli blueprints delete "${blueprint}" 2>/dev/null || true
     sudo composer-cli blueprints push "${BUILDDIR}/${blueprint_file}"
-    sudo composer-cli blueprints depsolve ${blueprint} 1>/dev/null
+    sudo composer-cli blueprints depsolve "${blueprint}" 1>/dev/null
 
-    if [ -n "$parent_version" ]; then
+    if [ -n "${parent_version}" ]; then
         title "Serving ${parent_blueprint} v${parent_version} container locally"
-        sudo podman rm -f ${parent_blueprint}-server 2>/dev/null || true
-        sudo podman rmi -f localhost/${parent_blueprint}:${parent_version} 2>/dev/null || true
-        imageid=$(cat ./${parent_blueprint}-${parent_version}-container.tar | sudo podman load | grep -o -P '(?<=sha256[@:])[a-z0-9]*')
-        sudo podman tag ${imageid} localhost/${parent_blueprint}:${parent_version}
-        sudo podman run -d --name=${parent_blueprint}-server -p 8080:8080 localhost/${parent_blueprint}:${parent_version}
+        sudo podman rm -f "${parent_blueprint}-server" 2>/dev/null || true
+        sudo podman rmi -f "localhost/${parent_blueprint}:${parent_version}" 2>/dev/null || true
+        imageid=$(cat < "./${parent_blueprint}-${parent_version}-container.tar" | sudo podman load | grep -o -P '(?<=sha256[@:])[a-z0-9]*')
```

Comment:
What's the reason for `cat FILE` => `cat < FILE` ?

Reply:
Oh, Shellcheck complains of [useless cat](https://www.shellcheck.net/wiki/SC2002), it makes the argument that instead of piping the output, use the second command to directly consume the file. So in this example, it would prefer we do `sudo podman --input ./${parent_blueprint}-${parent_version}-container.tar`. I was trying to keep changes more on the styling side and touch very little of the functionality. I don't mind going back and updating if we feel that would be better?

Reply:
I'll make a note of this and we can revisit this change